src/global_tracer.js

```diff
@@ -0,0 +1,67 @@
+import Tracer from './tracer';
+
+const noopTracer = new Tracer();
+let _globalTracer = null;
+
+// Allows direct importing/requiring of the global tracer:
+//
+// let globalTracer = require('opentracing/global');
+//      OR
+// import globalTracer from 'opentracing/global';
+//
+// Acts a bridge to the global tracer that can be safely called before the
+// global tracer is initialized. The purpose of the delegation is to avoid the
+// sometimes nearly intractible initialization order problems that can arise in
+// applications with a complex set of dependencies, while also avoiding the
+// case where
+class GlobalTracerDelegate extends Tracer {
+
+    _startSpan(...args) {
+        const tracer = _globalTracer || noopTracer;
+        return tracer._startSpan(...args);
```

Comment:
I think this should be calling public methods on the underlying implementation. Then if an application decides to depend on the global singleton, it can work with anything that implements public OT API, not just those implementations that extend it and only implement the `_` methods.